ingestion/src/metadata/ingestion/source/dashboard/metabase/metadata.py

```diff
@@ -136,6 +139,24 @@ def get_project_name(self, dashboard_details: Any) -> Optional[str]:
             )
         return None
 
+    def get_owner_ref(
+        self, dashboard_details: MetabaseDashboardDetails
+    ) -> Optional[EntityReference]:
+        """
+        Get dashboard owner from email
+        """
+        try:
+            if dashboard_details.creator_id:
+                owner_details = self.client.get_user_details(
+                    dashboard_details.creator_id
+                )
```

Comment:
Did you forget to add the changes made to the MetabaseClient? I'm not seeing a `get_user_details` method.

Reply:
ohh yes .Thanks for pointing that out